receiver/telemetrygeneratorreceiver/host_metrics_generator.go

```diff
@@ -15,56 +15,73 @@
+	{Name: "system.network.io", ValueMin: 0, ValueMax: 100000000, Type: "Sum", Unit: "By", Attributes: map[string]any{"device": "eth0", "direction": "receive"}},
+	{Name: "system.network.errors", ValueMin: 0, ValueMax: 1000, Type: "Sum", Unit: "{errors}", Attributes: map[string]any{"device": "eth0", "direction": "receive"}},
+	{Name: "system.network.errors", ValueMin: 0, ValueMax: 1000, Type: "Sum", Unit: "{errors}", Attributes: map[string]any{"device": "eth0", "direction": "transmit"}},
+	{Name: "system.network.dropped", ValueMin: 0, ValueMax: 1000, Type: "Sum", Unit: "{packets}", Attributes: map[string]any{"device": "eth0", "direction": "transmit"}},
+	{Name: "system.network.dropped", ValueMin: 0, ValueMax: 1000, Type: "Sum", Unit: "{packets}", Attributes: map[string]any{"device": "eth0", "direction": "receive"}},
+	{Name: "system.network.conntrack.max", ValueMin: 65536, ValueMax: 65536, Type: "Sum", Unit: "{entries}"},
+	{Name: "system.network.conntrack.count", ValueMin: 8, ValueMax: 64, Type: "Sum", Unit: "{entries}"},
+	{Name: "system.network.connections", ValueMin: 0, ValueMax: 64, Type: "Sum", Unit: "{connections}", Attributes: map[string]any{"protocol": "tcp", "state": "ESTABLISHED"}},
+	{Name: "system.network.connections", ValueMin: 0, ValueMax: 64, Type: "Sum", Unit: "{connections}", Attributes: map[string]any{"protocol": "tcp", "state": "LISTEN"}},
+	{Name: "system.cpu.time", ValueMin: 0, ValueMax: 10000, Type: "Sum", Unit: "s", Attributes: map[string]any{"state": "user", "cpu": "cpu0"}},
+	{Name: "system.cpu.time", ValueMin: 0, ValueMax: 10000, Type: "Sum", Unit: "s", Attributes: map[string]any{"state": "system", "cpu": "cpu0"}},
+	{Name: "system.cpu.time", ValueMin: 0, ValueMax: 10000, Type: "Sum", Unit: "s", Attributes: map[string]any{"state": "idle", "cpu": "cpu0"}},
+	{Name: "system.cpu.time", ValueMin: 0, ValueMax: 10000, Type: "Sum", Unit: "s", Attributes: map[string]any{"state": "interrupt", "cpu": "cpu0"}},
+	{Name: "system.cpu.time", ValueMin: 0, ValueMax: 10000, Type: "Sum", Unit: "s", Attributes: map[string]any{"state": "nice", "cpu": "cpu0"}},
+	{Name: "system.cpu.time", ValueMin: 0, ValueMax: 10000, Type: "Sum", Unit: "s", Attributes: map[string]any{"state": "softirq", "cpu": "cpu0"}},
+	{Name: "system.cpu.time", ValueMin: 0, ValueMax: 10000, Type: "Sum", Unit: "s", Attributes: map[string]any{"state": "steal", "cpu": "cpu0"}},
+	{Name: "system.cpu.time", ValueMin: 0, ValueMax: 10000, Type: "Sum", Unit: "s", Attributes: map[string]any{"state": "wait", "cpu": "cpu0"}},
 }
 
 func newHostMetricsGenerator(cfg GeneratorConfig, logger *zap.Logger) metricGenerator {
-	// ignore the config, which only contains the type and resource attributes
-	defaultConfig.ResourceAttributes = cfg.ResourceAttributes
-	g := newMetricsGenerator(defaultConfig, logger)
+	cfg, err := fillHostMetricsConfig(cfg, logger)
+	if err != nil {
+		logger.Error("Error filling host metrics config", zap.Error(err))
```

Comment:
is it ok to continue if we hit this error?

Reply:
I think so, fillHostMetricsConfig will skip the metrics that caused errors